### PR TITLE
Disable qodana rules in poly fills

### PR DESCRIPTION
**Pull request by ITaluone**
## IMPORTANT 

* [ ] If the PR touches the public API, the changes have been approved in a separate issue with the "api-approved" label.
* [ ] The code complies with the [Coding Guidelines for C#](https://www.csharpcodingguidelines.com/).
* [ ] The changes are covered by unit tests which follow the Arrange-Act-Assert syntax and the naming conventions such as is used [in these tests](../tree/develop/Tests/FluentAssertions.Equivalency.Specs/MemberMatchingSpecs.cs#L51-L430).
* [ ] If the PR adds a feature or fixes a bug, please update [the release notes](../tree/develop/docs/_pages/releases.md) with a functional description that explains what the change means to consumers of this library, which are published on the [website](https://fluentassertions.com/releases).
* [ ] If the PR changes the public API the changes needs to be included by running [AcceptApiChanges.ps1](../tree/develop/AcceptApiChanges.ps1) or [AcceptApiChanges.sh](../tree/develop/AcceptApiChanges.sh).
* [ ] If the PR affects [the documentation](../tree/develop/docs/_pages), please include your changes in this pull request so the documentation will appear on the [website](https://www.fluentassertions.com/introduction).
    * [ ] Please also run `./build.sh --target spellcheck` or `.\build.ps1 --target spellcheck` before pushing and check the good outcome

### CODE REVIEW

File: qodana.yaml [hunk 1]
```diff
@@ -29,3 +29,9 @@ exclude:
   - name: CheckNamespace
     paths:
       - Src\FluentAssertions\Polyfill
+  - name: UnusedMethodReturnValue.Global
```
Comment (dennisdoomen) [inline qodana.yaml:32]: 🤔 Isn't it better to exclude everything under `Polyfill`? See https://www.jetbrains.com/help/qodana/qodana-yaml.html#exclude-example

Comment (dennisdoomen) [reply qodana.yaml]: We don't care about any warnings in code added by the `Polyfill` library

**Review by jnyrup — APPROVED** [review]

File: qodana.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 version: "1.0"
 
 linter: jetbrains/qodana-dotnet:latest
-failThreshold: 19
+failThreshold: 20
```
Comment (dennisdoomen) [inline qodana.yaml:4]: Why did we have to increase the threshold?

Comment (IT-VBFK) [reply qodana.yaml:4]: Because previous PRs touched new files, which increased the qodana fails (for .netstandard 2.0+)

**Review by dennisdoomen — APPROVED** [review]

**Pull request merged**